src/charm.py

```diff
@@ -303,6 +302,19 @@ def logs_rules(self) -> Dict[str, Any]:
         """Return a list of logging rules."""
         return self._cos.logs_alerts
 
+    @property
+    def requested_receivers(self) -> set:
+        """Return a list of requested tracing receivers."""
+        protocols = self._cos.requested_protocols()
+        protocols.update(
+            [
```

Comment:
[nit/optimization] you can update directly with an iterator, no need to create a throwaway list for it.

`set().update(a for a in b)`